docs/resources/server/datamaps/builtin.md

```diff
@@ -170,5 +198,28 @@ Example:
 }
 ```
 
+## `neoforge:waxables`
```

Comment:
Again, like above, new line after the heading.

Reply:
```suggestion
## `neoforge:waxables`

```